critiquebrainz/data/dump_manager.py

```diff
@@ -221,18 +217,51 @@ def public(location, rotate=False):
         f.write(str(db.SCHEMA_VERSION))
 
     # BASE ARCHIVE
-    # Archiving stuff that is independent from licenses (users, licenses)
+    # Contains all license independent data (licenses, users)
+    base_archive_path = create_base_archive(dump_dir, temp_dir)
+    print(base_archive_path)
+
+    # 1. COMBINED
+    # Archiving all reviews (any license)
+    review_dump_path = create_reviews_archive(dump_dir, temp_dir)
+    print(review_dump_path)
+
+    # 2. SEPARATE
+    # Creating separate archives for each license
+    for license in db_license.list_licenses():
+        review_dump_path = create_reviews_archive(dump_dir, temp_dir, license_id=license['id'])
+        print(review_dump_path)
+
+    shutil.rmtree(temp_dir)  # Cleanup
+
+    if rotate:
+        print("Removing old dumps (except two latest)...")
+        remove_old_archives(location, "[0-9]+-[0-9]+", is_dir=True)
+
+    print("Done!")
+
+
+def create_base_archive(dump_dir, temp_dir):
+    """Creates a dump of all license-independent information: (users, license)."""
```

Comment:
This docstring should be improved. Add information about what this function takes as arguments and what it returns.